ui/design/src/components/ProfileSubCard/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Box, Text } from 'grommet';
+import { CSSProperties } from 'styled-components';
+
+import { ReleaseInfo } from '@akashaorg/ui-awf-typings';
+
+import Icon from '../Icon';
+import { MainAreaCardBox } from '../EntryCard/basic-card-box';
+import { BackgroundDiv } from '../SubtitleTextIcon/styled-subtitle-text-icon';
+
+export interface IProfileSubCardProps {
+  className?: string;
+  titleLabel: string;
+  ctaLabel: string;
+  apps: ReleaseInfo[];
+  onCTAClick?: () => void;
+  onAppClick?: (appId: string) => void;
+}
+
+const textStyles: CSSProperties = {
+  width: '190px',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  letterSpacing: '0.02rem',
+};
+
+const ProfileSubCard: React.FC<IProfileSubCardProps> = props => {
```

Comment:
ProfileSubcard? What is this?

Reply:
Can we also have a ProfileSubSubCard?

Reply:
the card under profile holding some records of the published apps

Reply:
so it's not a generic card. it's not even a `sub` of something...